docs/data-collection/tear-down.md

```diff
@@ -28,8 +28,16 @@
 - [ ] Give the participant the corresponding compensation for the participation and transportation.
 - [ ] Request a signed receipt of the amount of the financial compensation.
 
+!!! important "To boost efficiency, two people can work simultaneously on the following steps — one inside the scanning room and the other outside. They can follow the designated sections [inside the scanning room](#after-scan-inside-the-scanner-room) and [outside the scanning room](#after-scan-outside-the-scanner-room) in parallel."
+
 ## AFTER SCAN, inside the scanner room
 
+- [ ] Unplug the two cables connected to the ET (signal and power). Put those extremities aside far from the scanner.
+- [ ] Disconnect the GA and RB tubes.
+
+- [ ] **With someone outside in the control room**:
+    - [ ] Carefully extract the four cables one by one (the fiber and power of the ET, the tube of RB and the tube of the GA) back through the access tube. 
+  
 - [ ] **Carefully remove the infrared mirror**:
     - [ ] Enter again the scanner room with the plastic container of the mirror and leave it prepared on the bed.
     - [ ] Separate the mirror frame from the upper part of the head coil and lay it on the bed.
```

Comment:
This comment also replaces the following four lines (36-39 of the original text), GitHub does not allow me to modify them directly.

```suggestion
- [ ] The experimenter currently outside in the Control Room will enter the Scanner Room with the plastic bag of the mirror and a fresh pair of gloves and will leave them prepared on the bed.
- [ ] Remove the mirror frame from its rails mounted on the head coil and lay it on the bed.
- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.
- [ ] Immediately insert the infrared mirror back into its plastic pouch.
- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
- [ ] Re-attach the mirror to the coil's frame.
```